### PR TITLE
Switch `tmt.identifier` from using `fmf.log`

### PR DESCRIPTION
**Pull request by happz**

### CODE REVIEW

File: tests/unit/test_id.py [hunk 1]
```diff
@@ -37,10 +37,14 @@ def test_base(self):
         assert test.id is None
 
     def test_manually_add_id(self):
+        # TODO: it's really not possible to use fixtures with methods??
```
Comment (KwisatzHaderach) [inline tests/unit/test_id.py:40]: should be possible, however this is unittest and not pytest, so that might be the problem? :)

Comment (happz) [reply tests/unit/test_id.py:40]: It's still executed by pytest... Anyway, it's a problem for another day, I'll be glad to cut out one use of `fmf.log`, and I can address the fixture issue later.

**Review by thrix — APPROVED** [review]
LGTM

**Comment by thrix** [comment]
@happz some tests still fail, not sure if that is a flake. restarting

**Review by janhavlin — APPROVED** [review]

**Review by KwisatzHaderach — APPROVED** [review]

**Review by psss — APPROVED** [review]

**Pull request merged**